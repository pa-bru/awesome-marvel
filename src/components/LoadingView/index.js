import React, { Fragment } from 'react'

import CircularProgress from '@material-ui/core/CircularProgress'

import Center from 'components/Center'
import Head from 'components/Head'

const LoadingView = () => (
  <Fragment>
    <Head title="Loading..." />
    <Center>
      <CircularProgress color="secondary" />
    </Center>
  </Fragment>
)

export default LoadingView
