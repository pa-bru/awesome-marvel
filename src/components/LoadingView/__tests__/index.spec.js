import React from 'react'
import { shallow } from 'enzyme'
import LoadingView from '../'

describe('LoadingView', () => {
  it('should render correctly', () => {
    // When
    const tree = shallow(<LoadingView />)

    // Then
    expect(tree).toMatchSnapshot()
  })
})
