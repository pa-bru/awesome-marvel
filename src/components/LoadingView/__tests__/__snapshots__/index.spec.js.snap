// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingView should render correctly 1`] = `
<React.Fragment>
  <Head
    title="Loading..."
  />
  <Center>
    <WithStyles(CircularProgress)
      color="secondary"
    />
  </Center>
</React.Fragment>
`;
